feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/send/flow/AssetSendFlowViewModel.kt

```diff
@@ -38,4 +43,15 @@ class AssetSendFlowViewModel(
         val assePayload = AssetPayload(chainAsset.chainId, chainAsset.id)
         router.openSend(assePayload)
     }
+
+    override fun getPlaceholder(query: String, assets: List<Any>): PlaceholderModel? {
+        if (query.isEmpty() && assets.isEmpty()) {
+            return PlaceholderModel(
+                text = resourceManager.getString(R.string.assets_flow_placeholder),
+                imageRes = R.drawable.ic_no_search_results
+            )
```

Comment:
Should there by a call to action (Buy)? Or you plan to do it in a separate PR?

Reply:
I'll do it here